packages/components/src/__actions__/Button/v3/types.ts

```diff
@@ -0,0 +1,21 @@
+export type ButtonVariant = "primary" | "secondary" | "tertiary"
+
+export type ButtonSize = "small" | "medium" | "large"
```

Comment:
Convention is plurals :)
```suggestion
export type ButtonVariants = "primary" | "secondary" | "tertiary"

export type ButtonSizes = "small" | "medium" | "large"
```